phpunit.xml

```diff
@@ -4,6 +4,7 @@
          beStrictAboutChangesToGlobalState="true"
          beStrictAboutOutputDuringTests="true"
          colors="true"
+         stopOnFailure="true"
```

Comment:
Not sure this is a good idea. CI/CD uses this, and CI/CD should show all errors not just the first one. Also you can add it via the cli, but can't remove it via the cli, meaning this is something that can't be disabled without writing a whole new phpunit.xml and using that.